test/conformance/ingress/class.go

```diff
@@ -86,6 +87,7 @@ func verifyIngressWithAnnotations(ctx context.Context, t *testing.T, clients *te
 		},
 		OverrideIngressAnnotation(annotations),
 	)
+	t.Cleanup(cancel)
```

Comment:
It does this internally already and optionally returns it in case folks want to clean it up earlier (the way TestUpdate does).

Reply:
😠 
Done.